packages/lib/src/vis/hooks.ts

```diff
@@ -104,3 +104,36 @@ export function useCustomColors(
 
   return [colors, refCallback];
 }
+
+export function useRaycasterThreshold(
+  lineOrPoints: 'Line' | 'Points',
+  desiredThreshold: number
```

Comment:
I encapsulated the work on raycaster threshold made for the interaction on Scatter points (https://github.com/silx-kit/h5web/pull/1051 and https://github.com/silx-kit/h5web/pull/1064) to reuse it for the `DataCurve`.